### PR TITLE
bug_fixes

### PR DESCRIPTION
**Pull request by dharanisakary**
Hi Jon - Here are the changes for [In the Dragoon node editor's Units control, selecting "No Units" does not give red/green/yellow/blue feedback](https://trello.com/c/2NfB2GDh/129-in-the-dragoon-node-editor-s-units-control-selecting-no-units-does-not-give-red-green-yellow-blue-feedback) and  [Dragoon: The remove waveform button (AUTHOR mode) longer works](https://trello.com/c/QHOcSE9V/127-dragoon-the-remove-waveform-button-author-mode-longer-works). Please let me know for any changes.

### CODE REVIEW

File: www/index.php [hunk 1]
```diff
@@ -495,7 +495,8 @@
 					<div id="unitsQuestionMark" class="questionMark"></div>
 					<label id="selectUnitsControl">Units
 						<select id="selectUnits" data-dojo-type="dijit/form/Select">
-							<option value='defaultSelect'>No Units</option>
+							<option value='defaultSelect'>--Select--</option>
+							<option value='no-units'>No Units</option>
```
Comment (jonwetzel) [inline www/index.php]: This is causing "no-units" to get rendered on the node.  If there's no units, we should just display nothing in that spot.  Perhaps we can just make the value null or the empty string.

File: www/index.php [hunk 2]
```diff
@@ -495,7 +495,8 @@
 					<div id="unitsQuestionMark" class="questionMark"></div>
 					<label id="selectUnitsControl">Units
 						<select id="selectUnits" data-dojo-type="dijit/form/Select">
-							<option value='defaultSelect'>No Units</option>
+							<option value='defaultSelect'>--Select--</option>
```
Comment (jonwetzel) [inline www/index.php]: Can you add a "hidden" property to the -- Select -- option so it doesn't appear in the drop down?

**Comment by dharanisakary** [comment]
Hi Jon, I made few changes to fix units field. Can you please review the changes

**Comment by jonwetzel** [comment]
Looks great.  Merging to master!

**Pull request merged**